deps/build.jl

```diff
@@ -0,0 +1,28 @@
+using BinDeps
+using Compat
+
+@BinDeps.setup
+
+libmysqlclient = library_dependency("libmysqlclient",
+                                    alias=["libmysqlclient.so",
```

Comment:
On a side note, another option is to not use BinDeps at all, and ask the user to install MySQL him/herself. This is what we were doing earlier. I would suspect most people tend to connect to MySQL processes running on different DB servers.